src/modules/ROOT/pages/changelog/202303/01.json

```diff
@@ -0,0 +1,889 @@
+      ],
+      "why": [
+        "because to add support for Auth personalized screen.",
+        "Uploading the image will update the Auth background image, and the styles will supplement Auth screen's styles."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/446/commits/86d348700ad53aa9272d40bff97d0fdba03ee76e"
+      ]
+    },
+    {
+      "what": [
+        "new page block type for Geometry",
+        "Geometry page block renders a map within the rendering constraint from the Geometry configurator with points/markers and polygons.",
+        "Geometry configurator loads data from the configured feed sources."
+      ],
+      "why": [
+        "user can now configure a feed source for the Geometry which then loads the data in the record page, which is now rendered in markers and polygons."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/538/commits/82249fa67ea5d31656001d9f358d6e6abf5539a2" 
+      ]
+    },
+    {
+      "what": [
+        "new features to module Field configuration for styling image (fix height, fix width, rounded corners, border radius, background color, margin).",
+        "The applied style changes in module field setting now reflect on only file with single image and gallery mode selected as the view mode when viewing record page or record list page."
+      ],
+      "why": [
+        "to provide the user with more customization and styling option on viewing images."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/533/commits/ccd90d74d4945e96d21281c68bfe9f18bc394844"
+      ]
+    },
+    {
+      "what": [
+        "a record list page block in the Compose page builder with a dropdown field with three options for opening a record: in the same tab, in a new tab or in a modal.",
+        "The new added option is the `open record in a modal`.",
+        "When this option is selected and the user goes to a record list page or a page that displays record lists, he can click on a specific record and it will open as a modal.",
+        "From the modal, the user can do whatever he wants with the existing record (edit, delete, clone) or even add a new record."
+      ],
+      "why": [
+        "to make record entry a bit more intuitive, since the user will remain on the record list page while doing the changes."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/524/commits/2647e3b8d72afa31e79457f3cd985ae0b7fa31fc",
+        "https://github.com/cortezaproject/corteza/pull/524/commits/950d305a5a8fb79ee84599989a399af7d028f599"
+      ]
+    },
+    {
+      "what": [
+        "the ability to copy, clone, and paste page blocks in page builders."
+      ],
+      "why": [
+        "to make it possible to duplicate page blocks easily."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/582/commits/7f377109d0b46ade1b04857569622829f917321b"
+      ]
+    },
```

Comment:
Consider rewording this to be more in line with the page cloning counterpart.
Perhaps we could consider merging the two together.